pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -120,14 +120,22 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundHTTPRouteConfig(
 	return r
 }
 
+// IPv6 addresses are enclosed within square brackets
```

Comment:
Please update the comments, this is confusing

Reply:
Updated